pyheatmy/pyheatmy/time_series_multiperiodic.py

```diff
@@ -66,13 +72,22 @@ def plot(self, time_unit="s"):
 
         plt.plot(a[0], a[1])
         plt.title("Temperature profile")
-        plt.xlabel("time : " + str(time_unit))
+        plt.xlabel("date")
         plt.ylabel("temperature : °C")
         plt.show()
 
 
 # testing
 if __name__ == "__main__":
+    from datetime import datetime, timedelta
+
+    timestamp = 1485714600
+    date_0 = datetime.fromtimestamp(timestamp)
+    step = timedelta(days=2, hours=3)
+    dates = [date_0 + i * step for i in range(1000)]
+
     mp_ts = time_series_multiperiodic("multi_periodic")
-    mp_ts.create_multiperiodic_signal([10, 8, 9], [[5,'h'], [10, 'm'], [7, 'd']], time_step=15*NSECINMIN)
+    mp_ts.create_multiperiodic_signal(
+        [10, 5, 3], [[1, "y"], [2, "m"], [21, "d"]], dates, dt=2 * 86400 + 3 * 3600
```

Comment:
please use the environment variables defined in config.py NSECINDAY, etc Or add new environmental variables